spec/support/rswag_config.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class RswagConfig
+  def config
+    {
+      'public/openapi.json' => {
+        openapi: '3.0.3',
+        info: {
+          title: 'OpenAPI Docs',
+          version: '1.0'
+        },
+        paths: {},
+        servers: []
```

Comment:
```suggestion
        servers: [url: Settings.hostname]
```

Reply:
If this gets generated and committed, I think we'll need to include all of the environments. Either that, or this gets generated per environment? I think adding localhost + sandbox, staging and prod works. Curious what you think